v0/sources/canny/mapping.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "sourceKeys": "created",
+    "destKeys": ["timestamp", "originalTimestamp"]
+  },
+  {
+    "sourceKeys": "object",
+    "destKeys": "properties"
```

Comment:
You can exclude `author` or `voter` when mapping things from `object` 
example: https://github.com/rudderlabs/rudder-transformer/blob/a8560035932b96a18da0e37b98d3c25f149739ac/v0/destinations/vero/data/VeroIdentifyConfig.json#L16

Reply:
"excludes" is not working